.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,138 @@
+name: Bug Report
+description: Report a bug impacting VXLAN as Code collection
+title: Bug Report Title
+labels:
+  - bug
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for creating a bug issue to help us improve the VXLAN as Code collection. 
+        Please fill out the sections below to help us understand and reproduce the issue you are facing.
+
+  - type: markdown
+    attributes:
+      value: |
+        ### Community Note
+        Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
+        Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
+        If you are interested in working on this issue or have submitted a pull request, please leave a comment
+
+  - type: textarea
+    id: ansible-version
+    attributes:
+      label: Ansible Version
+      description: Please provide your Ansible version used when encountering this issue
+      placeholder: ansible --version
+      render: shell
+    validations:
+      required: true
+
+  - type: textarea
+    id: ansible-collection-versions
+    attributes:
+      label: Ansible Collection Versions
+      description: Please provide your Ansible Collection versions used when encountering this issue
+      placeholder: ansible-galaxy collection list
+      render: shell
+    validations:
+      required: true
+    
+  - type: textarea
+    id: ndfc-version
+    attributes:
+      label: Cisco NDFC Version
+      description: Please provide your Cisco NDFC version used when encountering this issue
+      render: shell
+    validations:
+      required: true
+
+  - type: textarea
+    id: nxos-version
+    attributes:
+      label: Cisco NX-OS Version
+      description: Please provide your Cisco NX-OS version used when encountering this issue
+      render: shell
+    validations:
+      required: false
+
+  - type: dropdown
+    id: role
+    attributes:
+      label: Which role is this issue related to?
+      multiple: false
+      options:
+        - cisco.nac_dc_vxlan.validate
+        - cisco.nac_dc_vxlan.create
```

Comment:
Looks better.  One small thing...

create, deploy and remote have the additional `dtc` as part of the namespace

`cisco.nac_dc_vxlan.dtc.create`

Reply:
You're right, my bust. I copied from role level part of readme quickly. Fixing and committing proper fqcn now.